buildSrc/src/main/kotlin/io/spine/internal/gradle/publish/CloudRepo.kt

```diff
@@ -26,9 +26,38 @@
 
 package io.spine.internal.gradle.publish
 
+import io.spine.internal.gradle.Repository
+
 /**
- * Popular names of Gradle configurations.
+ * CloudRepo Maven repository.
+ *
+ * There is a special treatment for this repository. Usually, fetching and pushing of artifacts
+ * is performed via the same URL. But it is not true for CloudRepo. Fetching is performed via
+ * public repository, and pushing via private one. Their URLs differ in `/public` infix.
  */
-internal object ConfigurationName {
-    const val archives = "archives"
+internal object CloudRepo {
+
+    private const val name = "CloudRepo"
+    private const val baseUrl = "https://spine.mycloudrepo.io/public/repositories"
+    private const val credentialsFile = "cloudrepo.properties"
+
+    /**
+     * Use this instance to depend on artifacts from this repository.
+     */
+    val reading = Repository(
```

Comment:
In the type-level docs above, you say "fetching" and "pushing". In the docs for this instance you say "depend on". Here you say "reading".

How about we have `CloudRepo.published`, so that its usage is `CloudRepo.published.releases`?

See cont'd below.